src/main/java/httpserver/http/response/Response.java

```diff
@@ -0,0 +1,34 @@
+package httpserver.http.response;
+
+import httpserver.http.ProtocolVersion;
+import httpserver.http.StatusCode;
+
+public class Response {
+    private final ProtocolVersion protocolVersion;
+    private final StatusCode statusCode;
+    private String headers;
+
+    public Response(ProtocolVersion protocolVersion, StatusCode statusCode) {
+        this.protocolVersion = protocolVersion;
+        this.statusCode = statusCode;
+    }
+
+    public Response(ProtocolVersion protocolVersion, StatusCode statusCode, String headers) {
```

Comment:
Think it would be cleaner if all fields were set in this constructor and the other constructor called this one - with `this(protocolVersion, statusCode, null);`

Reply:
Thanks, Matt. I'll change this around